src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int maxFigureSize;
+    private final double minDecimalPart;
```

Comment:
Let's create them as a constants. Init here, not in constructor.

Reply:
done